src/Identity/UI/src/Areas/Identity/Pages/V3/Account/Login.cshtml

```diff
@@ -42,7 +42,7 @@
                         <a asp-page="./Register" asp-route-returnUrl="@Model.ReturnUrl">Register as a new user</a>
                     </p>
                     <p>
-                        <button id="resend-confirmation" type="submit" asp-page-handler="SendVerificationEmail" class="btn-link" style="padding:0px;margin:0px;border:0px">Resend email confirmation</button>
```

Comment:
nit: Do we still have inline styles anywhere? We should avoid those.

Reply:
There's a few `class="btn btn-default"` on buttons, safe to just remove them everywhere?